packages/core-app-api/config.d.ts

```diff
@@ -65,6 +65,26 @@ export interface Config {
         }>;
       }>;
     };
+    /**
+     * Running the frontend app with https
+     */
+    https?: {
+      /**
+       * Parent object containing certificate and the private key
+       */
+      certificate?: {
+        /**
+         * Https Certificate private key. Use $file to load in a file
+         * @visibility secret
+         */
+        key?: string;
```

Comment:
This and the cert should probably be required, so that it complains if you only try to provide one of them. The certificate block itself is already optional anyway